gnome/gstreamer1-gst-plugins-bad/Portfile

```diff
@@ -37,35 +37,18 @@ depends_build-append        port:pkgconfig \
 
 depends_lib                 port:gstreamer1-gst-plugins-base \
                             port:gobject-introspection \
```

Comment:
I’ll need to check this one but think I remember gobject-introspection getting linked meaning it will be needed at runtime.